gcapi/gcapi.py

```diff
@@ -109,8 +112,9 @@ def __init__(self, client):
             setattr(self, k, api(self._client))
 
     def _verify_against_schema(self, value):
-        if self.json_schema is not None:
-            self.json_schema.validate(value)
+        schema = self.validation_schemas.get("GET")
+        if schema is not None:
+            schema.validate(value)
```

Comment:
Will all these schemas work if the schema is `None`, as in, a nullable field? I guess the schema itself is never null, only the attributes are.

Reply:
A schema has to be a Draft7Validator instance that contains the schema. The schema was already adapted for that, see for example here:

https://github.com/DIAGNijmegen/rse-gcapi/blob/b00fb9f849fc8a39d6a90ff701addc7f2ea1f917/gcapi/schemas/answer.json#L219-L234

I only made sure that these schemas are only applied to certain types of requests - POST in this case. I did not write a PATCH jsonschema. So those go out unchecked right now.